packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -431,6 +486,139 @@ function RemoveLinkFormatButton() {
   )
 }
 
+export interface TableButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  readonly icon: IconNames | SVGIcon
+  readonly iconActive?: IconNames | SVGIcon
+  readonly active?: boolean
+}
+
+export const BaseTableButton = forwardRef<HTMLButtonElement, TableButtonProps>(
+  function BaseTableButton({icon, iconActive, active, ...props}, ref) {
+    icon = iconActive && active ? iconActive : icon
+    return (
+      <button
+        style={{
+          border: active ? 'blue 1px solid' : '',
+          fontSize: 16,
```

Comment:
Don't you need a unit here? If default is `px` I wonder if this is the right way to do it since on high-res screens `px` are not being scaled

Reply:
We already have a ticket for that. https://wepublish.atlassian.net/browse/WEP-118